report/src/introduction/agile_approach.tex

```diff
@@ -33,18 +31,22 @@ \subsubsection*{Scrum Artifacts and Tools}
 As the stakeholders provide feedback, items in the backlog should be prioritized accordingly.
 All these items contribute to achieving the \textbf{product goal}.
 The backlog items are \textbf{user stories}.
-These stories represent functionality the stakeholder would like to see in the system.
+
+User stories represent functionality the stakeholder would like to see in the system.
 For this reason, they must include \textit{who} it is for, \textit{what} should be done, and \textit{why} they want it.
 This is captured in the general "\texttt{As a X, I want Y, such that Z.}" format for user stories.
 A \textbf{sprint} is a short period of time, during which the team works on a subset of the product backlog, captured in a \textbf{sprint backlog}.
 This defines the \textit{what} of the sprint.
 The items in the sprint backlog are chosen based on a \textbf{sprint goal}, defining the \textit{why} of the sprint.
-The sprint also has an actionable plan for delivering the increment, representing the \textit{how}.
+
+
+The sprint has an actionable plan for delivering the increment, representing the \textit{how}. \todo{what is how? It is not mentioned anywhere else.}
```

Comment:
`What`, `how`, and `why` are referenced throughout the three previous sentences to denote the functions that the aspects presented have. To say that a sprint backlog is the `what` means that it denotes what should be done. Similarly, saying that the sprint goal denotes the `why` is to say that the spring goal denotes why the sprint is being undertaken.
And finally, `how` would then be represented by the actionable plan to deliver the increment.
That is to say, we know `what` we are doing, we know `why` we are doing it, and we know `how` to do it.